lib/IMAP/FolderMapper.php

```diff
@@ -62,12 +62,24 @@ public function getFolders(Account $account, Horde_Imap_Client_Socket $client,
 			'special_use' => true,
 		]);
 
-		return array_filter(array_map(function (array $mailbox) use ($account) {
+		return array_filter(array_map(function (array $mailbox) use ($account, $client) {
 			if (in_array($mailbox['mailbox']->utf8, self::DOVECOT_SIEVE_FOLDERS, true)) {
 				// This is a special folder that must not be shown
 				return null;
 			}
 
+			try {
+				$client->status($mailbox["mailbox"]);
+			} catch (Horde_Imap_Client_Exception $e) {
+				// ignore folders which cause errors on access
+				// (i.e. server-side system I/O errors)
+				if (in_array($e->getCode(), [
+					Horde_Imap_Client_Exception::UNSPECIFIED,
+				])) {
```

Comment:
please pass `true` as third param of `in_array`. It's unlikely that php would do weird casting with this particular code but static analysis will thank us for this.

Ref https://www.php.net/manual/en/function.in-array.php

Reply:
Done.